src/textual/geometry.py

```diff
@@ -87,24 +89,28 @@ def area(self) -> int:
 
     @property
     def region(self) -> Region:
-        """Get a region of the same size."""
+        """Get a region of the same size.
+        
+        Returns:
+            Region: a `Region` object of the same size, with origin (0, 0).
+        """
         width, height = self
         return Region(0, 0, width, height)
 
     def contains(self, x: int, y: int) -> bool:
         """Check if a point is in the size.
 
         Args:
-            x (int): X coordinate (column)
-            y (int): Y coordinate (row)
+            x (int): X-coordinate (column)
+            y (int): Y-coordinate (row)
 
         Returns:
             bool: True if the point is within the region.
         """
         width, height = self
         return width > x >= 0 and height > y >= 0
 
-    def contains_point(self, point: tuple[int, int]) -> bool:
+    def contains_point(self, point: TwoIntTuple) -> bool:
```

Comment:
Not overly keen on `TwoIntTuple` as I don't think it is more descriptive than `tuple[int, int]`.